### PR TITLE
[W5.11][W14-A4] Ngo Phuong Thao

### PR DESCRIPTION
**Pull request by Sisyphus25**
- Change find command to be case insensitive

### CODE REVIEW

**Review by Sisyphus25 — COMMENTED** [review]
The commit is supposed to make the find command case-INSENSITIVE, not sensitive.

**Review by tzerbin — COMMENTED** [review]
Refer comment below

File: test/input.txt [hunk 1]
```diff
@@ -96,7 +96,7 @@
   find bet
   # does not match if none have keyword
   find 23912039120
-  # matching should be case-sensitive
+  # matching should be case-insensitive
```
Comment (tzerbin) [inline test/input.txt]: Would have been better if the test cases added consist of matching that requires the conversion of lower-upper cases

**Review by kowshik-sundararajan — COMMENTED** [review]
Good work! Please close the PR.

**Pull request closed without merging**